config/samples/backends/bases/openstack/kustomization.yaml

```diff
@@ -1,2 +1,24 @@
 resources:
-  - openstack.yaml
+  - https://raw.githubusercontent.com/openstack-k8s-operators/openstack-operator/main/config/samples/core_v1beta1_openstackcontrolplane_galera_network_isolation.yaml
```

Comment:
Ooh, this is good! I like the way this references the openstack-operator's sample, rather than the original hard coded one in the cinder-operator which is likely to drift out of sync with the rest of the operators.

Reply:
And this gets even better when you look at the glance using cinder samples I wrote, because they can reference the whole cinder LVM iscsi sample and only change the glance part: https://github.com/openstack-k8s-operators/glance-operator/blob/main/config/samples/backends/cinder/lvm-iscsi/kustomization.yaml